src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function sortRowsByNumber(rows, childSelector) {
+  let result = [];
+  const childNomer = childSelector + 1;
```

Comment:
The variable name 'childNomer' seems to be a typo. It should likely be 'childNumber' to reflect its purpose more clearly.